server_admin/topics/authentication/webauthn.adoc

```diff
@@ -157,7 +155,8 @@ The appropriate method to register a WebAuthn authenticator depends on if the us
 
 New user::
 
-A new user carries out the following operations :
+In case that `WebAuthn Register` required action is set as `Default Action` in a realm, a new users will be always required to
```

Comment:
If the `WebAuthn Register` required action is set as `Default Action` in a realm, new users are required to
set up the WebAuthn security key after the first successful login. A new user carries out the following operations :